Sources/Rendering/Core/Camera/index.js

```diff
@@ -338,6 +338,16 @@ function vtkCamera(publicAPI, model) {
     publicAPI.modified();
   };
 
+  publicAPI.translate = (x, y, z) => {
+    const offset = [x, y, z];
+
+    vtkMath.add(model.position, offset, newPosition);
+    vtkMath.add(model.focalPoint, offset, newFocalPoint);
+
+    publicAPI.setPosition(...newPosition);
+    publicAPI.setFocalPoint(...newFocalPoint);
```

Comment:
the idea of creating translate is to avoid calling modified and compute distance twice, but only once.